server/src/immich/api-v1/asset/asset.controller.ts

```diff
@@ -125,12 +124,11 @@ export class AssetController {
   @ApiOkResponse({ content: { 'application/octet-stream': { schema: { type: 'string', format: 'binary' } } } })
   serveFile(
     @AuthUser() authUser: AuthUserDto,
-    @Headers() headers: Record<string, string>,
-    @Response({ passthrough: true }) res: Res,
+    @Response() res: Res,
```

Comment:
Since we remove passthrough, have you verified that throwing exceptions work correctly still?